specification/ml/get_overall_buckets/MlGetOverallBucketsRequest.ts

```diff
@@ -139,7 +139,7 @@ export interface Request extends RequestBase {
     /**
      * Refer to the description for the `overall_score` query parameter.
      */
-    overall_score?: double | string
+    overall_score?: double
```

Comment:
While this one isn't a query parameter, it feels like a straight-up bug.

Parsing is done in https://github.com/elastic/elasticsearch/blob/eb3bc2f2929209461802884ea01f77d3b3fcf96a/x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/GetOverallBucketsAction.java#L73